openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -57,6 +60,7 @@ class CoursewareInformation(DeveloperErrorViewMixin, RetrieveAPIView):
         * enrollment: Enrollment status of authenticated user
             * mode: `audit`, `verified`, etc
             * is_active: boolean
+        * can_view: Whether the user can view the course
```

Comment:
Thoughts on an alternative name like `user_has_access`?

Reply:
Yeah, that sounds clearer. 👍